src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -169,31 +169,20 @@ public static class DnnCatalog
                 ReuseValidationSetBottleneckCachedValues = reuseValidationSetBottleneckCachedValues
             };
 
-            if (!File.Exists(options.ModelLocation))
+            if (options.Arch == Architecture.InceptionV3)
             {
-                if (options.Arch == Architecture.InceptionV3)
-                {
-                    var baseGitPath = @"https://raw.githubusercontent.com/SciSharp/TensorFlow.NET/master/graph/InceptionV3.meta";
-                    using (WebClient client = new WebClient())
-                    {
-                        client.DownloadFile(new Uri($"{baseGitPath}"), @"InceptionV3.meta");
-                    }
+                var baseGitPath = @"https://raw.githubusercontent.com/SciSharp/TensorFlow.NET/master/graph/InceptionV3.meta";
```

Comment:
>ar baseGitPath = @"https://raw.githubusercontent.com/SciSharp/TensorFlow.NET/master/graph/InceptionV3.meta"; [](start = 17, length = 108)

It seems we don't have a test for Inception V3 and I will prefer we add one since this piece of code seems to be doing more than just downloading meta file, it is also download check point file. It should be pretty straight forward .. just copy paste,.